src/main/java/com/aws/greengrass/clientdevices/auth/metrics/MetricsEmitter.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.clientdevices.auth.metrics;
+
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+import javax.inject.Inject;
+
+public class MetricsEmitter {
+    protected ScheduledFuture<?> future;
```

Comment:
nit: why protected? can we lower to private?

Reply:
Had it in a separate file earlier but it can be private now. I'm not sure what you mean by the config stuff though

Reply:
ah I misunderstood it the first time around, I'll go address the config stuff here